lib/vyasa/medium.ex

```diff
@@ -137,17 +136,18 @@ defmodule Vyasa.Medium do
   def get_event!(id), do: Repo.get!(Event, id)
 
   def get_event_by_order!(%Event{origin: origin, voice_id: v_id}, order) do
-    #TODO merge Sangh Filters to fix -1 order case for origin backwards operators
-    (from e in Event,
+    # TODO merge Sangh Filters to fix -1 order case for origin backwards operators
```

Comment:
@ks0m1c reminder on this TODO

Reply:
is this still relevant?

Reply:
this was actually built out for the admin view idea before Repo.Paginated.all(opts) was merged. with Vyasa.Repo.Paginated module filter is expressed as

[filter: String.to_integer(cursor) |> DateTime.from_unix!(:millisecond), sort_attribute: :origin]